spatialos/UnrealGDK
# Bugfix/Fix case for Mac

#### Description
Fix case for Mac, keep it consistent with Windows.

#### Tests
How can this be verified by QA?
Execute the Setup.sh on Mac and check if there is a directory with the name "WorkerSDK" inside  "SpatialGDK/Source/SpatialGDK/Public/"
